NuGet.config

```diff
@@ -1,6 +1,13 @@
 <?xml version="1.0" encoding="utf-8"?>
 <!-- Licensed to the .NET Foundation under one or more agreements. The .NET Foundation licenses this file to you under the MIT license. See the LICENSE file in the project root for more information. -->
 <configuration>
+  <packageRestore>
+    <!-- NuGet Restore was observed to cause performance problems for in-IDE builds,
```

Comment:
Link issues that need to be fixed in order to get this workaround removed.

Reply:
Linked